src/components/swapv2/LimitOrder/useValidateInputError.tsx

```diff
@@ -39,10 +44,23 @@ const useValidateInputError = ({
 
       const remainBalance = parsedActiveOrderMakingAmount ? balance?.subtract(parsedActiveOrderMakingAmount) : undefined
       if (parseInputAmount && remainBalance?.lessThan(parseInputAmount)) {
-        const formatNum = formatNumberWithPrecisionRange(parseFloat(remainBalance.toFixed(3)), 0, 10)
-        return t`You don't have sufficient ${currencyIn?.symbol} balance. After your active orders, you have ${
-          Number(formatNum) !== 0 ? '~' : ''
-        }${formatNum} ${currencyIn?.symbol} left.`
+        const formatNum = formatDisplayNumber(remainBalance, {
+          style: 'currency',
+          fractionDigits: 6,
+          allowNegative: true,
```

Comment:
```suggestion
          allowNegative: true,
          allowZero: false,
```

Reply:
why would this place need allow negative?

Reply:
can be negative. 
can be zero

Reply:
This flag is for allowing display zero or negative or fallback with `--`. I saw this formatNum is comparing with zero and display it as ~ below so this num is not allow to show zero?

Reply:
Might be renaming to `allowDisplayNegative` & `allowDisplayZero` will be better to read

Reply:
I dont want to use allowZero for this case
Just want to add prefix `~`
`~` amount if (amount != 0) else amount

Reply:
got it